README.md

```diff
@@ -25,12 +25,12 @@ Go to the [Contributing guide](CONTRIBUTING.md) to learn how to get involved.
 The same codebase is used to build images for **Red Hat Advanced Cluster Management for Kubernetes** (from `release-*` branches) and **multicluster engine for Kubernetes** (from `backplane-*` branches). The build system fast-forwards between branches to keep the content in sync. Pull requests should only be opened against the first branch in each line listed below. The arrow represents an automatic fast-forwarding of commits from one branch to the next.
 
 ```
-main → release-2.14 → backplane-2.9
+main → release-2.15 → backplane-2.10
+release-2.14 → backplane-2.9
 release-2.13 → backplane-2.8
 release-2.12 → backplane-2.7
 release-2.11 → backplane-2.6
 release-2.10 → backplane-2.5
-release-2.9 → backplane-2.4
```

Comment:
For future reference, depending on release timing, it's not always the case that the oldest drops off the list.